README.md

```diff
@@ -41,6 +41,11 @@ In case you need CLI support for automation purposes, the plugin is also compati
 $ npm install verdaccio-github-oauth-ui
 ```
 
+#### Compatibility
+
+This plugin is currently only compatible with Verdaccio 3.x.
+This plugin supports Node versions 0.6.x - 0.10.x.
```

Comment:
I'm confused about this version range. Is this semantically the same as in package.engines?

Reply:
Fixed typo to match package.engines